components/CreateTodo.js

```diff
@@ -0,0 +1,118 @@
+
+import Toast from "./Toast.js";
+
+const CreateTodo = () => {
+  const buttonCreate = document.querySelector('#buttonCreate');
+  const createTodo = document.querySelector('.create-todo');
```

Comment:
can we also target createTodo using an id to be consistant along with other dom targets?